website/app/initializers/showdown-extensions.js

```diff
@@ -92,7 +93,11 @@ export function initialize(/* application */) {
           language = 'shell';
         }
 
-        let highlightedCodeBlock = Prism.highlight(codeblock, Prism.languages[language], language) + end;
+        if (Prism.languages[language]) {
```

Comment:
There is a problem of highlighting when the page loads and you change to the "Code" tab, vs when you load directly the "Code" tab. I suspect is related to this, because for a split second you see the correct syntax highlighting, but then it disappears.

https://github.com/hashicorp/design-system/assets/686239/aca95dda-803e-4083-80e6-7066438fdab2

Reply:
Yes, it's an issue caused by the showdown extension. The change highlighted above makes it work in some instances and prevents an Ember error that stops the page from rendering, but it still has pitfalls. I couldn't find a solution last week – I'll keep digging, but not an easy one to crack.

Reply:
A different tool would certainly fix the issue, but it would also require additional time.
The only solution I could find was to [add handlebars (and weirdly another library that it depends on 🙃 )](https://github.com/hashicorp/design-system/pull/1687/commits/2a8c1b1a0c0051ffcf46da02c37644a996e105d2) to the `CodeBlock` component.

You can see it in action below. As with the line highlight, we'd need to merge the docs branch into the component to see it working, but I thought I leave a comment here to address the question at source.
https://github.com/hashicorp/design-system/assets/788096/d3108e4c-2e1b-4343-b9dc-41794dc5ae57